support/docker/production/Dockerfile.stretch

```diff
@@ -1,4 +1,5 @@
-FROM node:10-stretch-slim
+# use buster for new enough ffmpeg
```

Comment:
Please remove the comment, and instead specifiy that kind of extraneous information in the body of your PR.

Reply:
I added it as a comment because the file is named stretch